solution Ihor Sobko (react_static-list-of-todos)

igaryok opened the pull request:
-[DEMO Link](https://igaryok.github.io/react_static-list-of-todos/)

irenhh commented on src/App.js:
```diff
@@ -3,16 +3,27 @@ import './App.css';
+}
 
 function App() {
+  const fullList = todosWithUser(todos, users);
```
you can write this outside the function App and get rid of `return`

igaryok replied:
> you can write this outside the function App and get rid of `return`

refacrored

irenhh commented on src/components/TodoList/TodoList.js:
```diff
@@ -0,0 +1,25 @@
+  );
+}
+
+export default TodoList;
```
move this to the end of the document

igaryok replied:
> move this to the end of the document

fix

irenhh commented on src/components/User/User.js:
```diff
@@ -0,0 +1,23 @@
+  return (
+    <>
+      <span className="li-user">User</span>
+      &nbsp;:&nbsp;
```
use template string here

igaryok replied:
> use template string here

refacrored

igaryok commented on src/App.js:
```diff
@@ -3,16 +3,27 @@ import './App.css';
+import TodoList from './components/TodoList/TodoList';
+
+function todosWithUser(listTasks, listUsers) {
+  return listTasks.map((item) => {
```
> rewrite this without `return`

refactored

igaryok commented:
> https://prnt.sc/p43vht add space before the number -- what about this one?
> and first two comments about code weren't fully fixed

1) added space before number
2) the first two coments - the first comment - i refactored funcrion without return, the second comment - i writeed const outside function. But i don't understand about this - `and get rid of return` - i can't delete return from fubction App()

irenhh commented:
have you opened the page?
you need to review it by yourself firstly

igaryok commented:
> have you opened the page?
> you need to review it by yourself firstly

sorry, i forgot to do deploy - fixed

irenhh commented on src/components/TodoItem/TodoItem.js:
```diff
@@ -0,0 +1,35 @@
+import './TodoItem.css';
+import User from '../User/User';
+
+function TodoItem(props) {
```
add destructuring

igaryok replied:
> add destructuring

refactored

irenhh replied:
 you haven't changed anything

igaryok replied:
>  you haven't changed anything

fixed

irenhh commented on src/components/TodoItem/TodoItem.js:
```diff
@@ -0,0 +1,35 @@
+function TodoItem(props) {
+  let classItem = 'list-group-item';
+
+  if (props.todo.completed) {
```
use ternary operator

igaryok replied:
> use ternary operator

fixed

irenhh commented on src/components/TodoList/TodoList.js:
```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+import TodoItem from '../TodoItem/TodoItem';
+
+function TodoList(props) {
```
rewrite without return

igaryok replied:
> rewrite without return

fixed

irenhh commented on src/components/User/User.js:
```diff
@@ -0,0 +1,19 @@
+import PropTypes from 'prop-types';
+
+function User({ name, email }) {
+  return (
```
rewrite without return

igaryok replied:
> rewrite without return

fixed
